cmd/process-agent/app/check.go

```diff
@@ -28,17 +30,24 @@ import (
 	"github.com/DataDog/datadog-agent/pkg/workloadmeta"
 )
 
+func init() {
+	CheckCmd.Flags().BoolVar(&checkOutputJSON, "json", false, "Output check results in JSON")
```

Comment:
Do we plan to add other output types? Boolean might not be the best choice if so.

For context, many CLIs use a `-o <format>` flag, so we might want an enum for the output format. The other consideration is that the core agent check has two flags to control the output, `--json` and `--table`, and `--json` supersedes `--table` (so that is similar to what we're doing here)

Reply:
I like the `-o` suggestion

Reply:
Let's stick with `--json` bool for now, may not be the most extensible but there is consistency with status command in the core agent. Other formats could be interesting, but having one machine-readable format can be sufficient for now.

Reply:
For the record, the core agent also has `--table`, which we might want to add in the future. Their solution to the problem is having `--json` take precedence over `--table`